src/client/src/components/pages/courses/addCourses/staticData.js

```diff
@@ -24,14 +24,14 @@ export const fields = [
     [
         {
             tag: "Input",
-            label: "Course start",
+            label: "Course Start",
             name: "course_start",
             width: "20rem",
             type: "date",
         },
         {
             tag: "Input",
-            label: "Course end",
+            label: "Course End",
```

Comment:
It might not be necessary here, but maybe this will help in the future:
There's a css property, `text-transform`, that you can use to automatically capitalize text inside an element.  It's really useful if you don't have hard-coded strings, so you can always capitalize or uppercase data that comes from a server or from user input:

https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform

Reply:
I'm not sure if it notifies you since I'm commenting on a merged PR so I'll just @ you, @AsalaKM